change/@ni-nimble-components-083cf44c-68b4-43f6-b689-efd81c8228b4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Since `buttonFillActivePrimaryColor` and `buttonFillAccentActiveColor` were previously exported design tokens, it seems like that makes this a `major` change (and we should note removing those tokens in the change notes).